### PR TITLE
getCoinCombination

### PR DESCRIPTION
**Pull request by joanna-konopacka**

### CODE REVIEW

File: src/getCoinCombination.test.js [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
+  });
+
+  it('should return [2, 1, 1, 0] for 17 cents', () => {
+    expect(getCoinCombination(17)).toEqual([2, 1, 1, 0]);
+  });
+
+  it('should return [0, 0, 0, 2] for 50 cents', () => {
+    expect(getCoinCombination(50)).toEqual([0, 0, 0, 2]);
+  });
+
```
Comment (vlasiuksasha) [inline src/getCoinCombination.test.js:24]: Test case for negative inputs is missing. It's important to test how your function behaves with unexpected inputs.

File: src/getCoinCombination.test.js [hunk 2]
```diff
@@ -1,10 +1,33 @@
...
+  it('should return [0, 0, 0, 0] for 0 cents', () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
+
+  it('should handle large amounts (e.g., 1234 cents)', () => {
+    expect(getCoinCombination(1234)).toEqual([4, 1, 0, 48]);
```
Comment (vlasiuksasha) [inline src/getCoinCombination.test.js:30]: Test case for non-integer inputs is missing. Be sure to test all edge cases.

File: src/getCoinCombination.test.js [hunk 3]
```diff
@@ -1,10 +1,33 @@
 'use strict';
 
+
 describe('getCoinCombination', () => {
-  const { getCoinCombination } = require('./getCoinCombination');
```
Comment (vlasiuksasha) [inline src/getCoinCombination.test.js:4]: The function 'getCoinCombination' is not imported. Make sure to import this function before invoking it.

**Review by vlasiuksasha — CHANGES_REQUESTED** [review]
check the comments

File: src/getCoinCombination.test.js [hunk 4]
```diff
@@ -1,10 +1,42 @@
-'use strict';
+"use strict";
 
-describe('getCoinCombination', () => {
-  const { getCoinCombination } = require('./getCoinCombination');
+describe("getCoinCombination", () => {
+  const { getCoinCombination } = require("./getCoinCombination");
+  describe("getCoinCombination", () => {
```
Comment (vlasiuksasha) [inline src/getCoinCombination.test.js:5]: The use of a nested describe block with the same name 'getCoinCombination' is redundant. Consider removing the nested describe block to make the tests more readable.

**Review by vlasiuksasha — CHANGES_REQUESTED** [review]
check the comment

**Review by vlasiuksasha — APPROVED** [review]
